java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -279,4 +279,21 @@ CompletableFuture<Double> zaddIncr(
      * }</pre>
      */
     CompletableFuture<Double> zscore(String key, String member);
+
+    /**
+     * Calculates and stores the set difference between the first sorted set and all the successive
+     * sorted sets at <code>destkey</code>, overwriting it if it already exists. Non-existent keys are
+     * treated as empty sets.
+     *
+     * @see <a href="https://redis.io/commands/zdiffstore/">redis.io</a> for more details.
+     * @param destKey Destination key for the resulting sorted set.
+     * @param keys The keys of the sorted sets.
+     * @return The number of members in the resulting sorted set at <code>destKey</code>.
+     * @example
+     *     <pre>{@code
+     * Long payload = client.zdiffstore("mySortedSet", new String[] {"key1", "key2"}).get();
```

Comment:
I think it would be helpful to include an example with 2 or 3 sets to compare.

Reply:
What do you mean? Just add one more set to the argument list?